packages/adminrouter/extra/src/includes/server/open/agent.conf

```diff
@@ -7,6 +7,13 @@ location /pkgpanda/ {
     proxy_redirect http://$http_host/ /pkgpanda/;
 }
 
+# Group: System
+# Description: Node and cluster checks
+location /system/checks/v1 {
+    include includes/proxy-headers.conf;
+    proxy_pass http://dcos_checks_api;
+}
```

Comment:
Is there a reason why we also add it the same endpoint in `packages/adminrouter/extra/src/includes/server/ee/common.conf` in the EE PR? Can we deduplicate this please?